[fix #10] [mozmeao/infra#1104] Clean up oregon-a admin yaml.

jpetto opened the pull request.

jpetto committed:
[fix #10] [mozmeao/infra#1104] Clean up oregon-a admin yaml.

jgmize commented on oregon-a/basket-admin/web-deploy.yaml:
```diff
@@ -48,281 +37,168 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-admin-v34-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-a/basket-admin/web-deploy.yaml:
```diff
@@ -48,281 +37,168 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-admin-v34-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-a/basket-admin/web-deploy.yaml:
```diff
@@ -48,281 +37,168 @@ spec:
-              key: wsgi-num-workers
-              name: basket-admin-v34-env
-        image: 127.0.0.1:5555/basket-admin:v34
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-a/basket-admin/web-deploy.yaml:
```diff
@@ -48,281 +37,168 @@ spec:
-              name: basket-admin-v34-env
-        image: 127.0.0.1:5555/basket-admin:v34
+          value: 4
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jpetto committed:
Review fixes:

- remove superfluous data from ns.yaml
- quote all numeric env var values
- use full image hash
